lib/kwtool_web/controllers/api/v1/fallback_controller.ex

```diff
@@ -10,4 +10,11 @@ defmodule KwtoolWeb.Api.V1.FallbackController do
     |> put_view(ErrorView)
     |> render("error.json", %{code: :validation_error, changeset: changeset})
   end
+
+  def call(conn, {:error, :not_found}) do
+    conn
+    |> put_status(:not_found)
+    |> put_view(ErrorView)
+    |> render("404.json", %{code: :not_found, message: "Keyword cannot be found."})
```

Comment:
This fallback controller is generic and may be used by other resource controllers. So we should not fix the error message to "Keyword".

How about we support the resource name and fallback to default value when the name is not given?
```elixir
 def call(conn, {:error, :not_found, resource}) when is_binary(resource) do
    conn
    |> put_status(:not_found)
    |> put_view(ErrorView)
    |> render("404.json", %{code: :not_found, message: "#{resource} cannot be found."})
 end
 
 def call(conn, {:error, :not_found}) do
    conn
    |> put_status(:not_found)
    |> put_view(ErrorView)
    |> render("404.json", %{code: :not_found, message: "Resource cannot be found."})
 end
```

Reply:
yup, very cool suggestion, i updated here: https://github.com/byhbt/kwtool/pull/48/commits/f217c7e9f890e83c9f51549023d5ee023ac7dc35